### PR TITLE
jakartaize JCA specification document, section 55 - 94

### PR DESCRIPTION
**Pull request by bbulk**

### CODE REVIEW

File: spec/src/main/asciidoc/Connectors.adoc [hunk 1]
```diff
@@ -4655,12 +4641,12 @@ services that enable transactional context to be propagated across
...
 directly to the application components.
 
-The connector architecture defines a
+The Jakarta Connector Architecture defines a
 transaction management contract between an application server and a
 resource adapter and its underlying resource manager. The transaction
 management contract has two parts, depending on the type of transaction:
 
-a JTA _javax.transaction.xa.XAResource_ based
+a Jakarta Transactions _jakarta.transaction.xa.XAResource_ based
```
Comment (smillidge) [inline spec/src/main/asciidoc/Connectors.adoc:4649]: I think XA resource is a JDK api

Comment (bbulk) [reply spec/src/main/asciidoc/Connectors.adoc:4649]: According to the docs
https://javaee.github.io/javaee-spec/javadocs/
I think it's part of Java EE

File: spec/src/main/asciidoc/Connectors.adoc [hunk 2]
```diff
@@ -3248,7 +3241,7 @@ javax.resource.spi.ManagedConnectionFactory
  public ManagedConnection
 createManagedConnection(
 
- javax.security.auth.Subject subject,
+ jakarta.security.auth.Subject subject,
```
[the remark this answers is not in the document]
Comment (bbulk) [reply spec/src/main/asciidoc/Connectors.adoc]: Correct, I've changed it.

File: spec/src/main/asciidoc/Connectors.adoc [hunk 3]
```diff
@@ -3103,20 +3095,20 @@ An implementation class for a connection
 factory interface must implement _java.io.Serializable_ . This enables a
 connection factory instance to be stored in the JNDI naming environment.
 A connection factory implementation class must implement the interface
-_javax.resource.Referenceable_ . Note that the
-_javax.resource.Referenceable_ interface extends the
-_javax.naming.Referenceable_ interface. Refer to section
-link:conn.html#a5327[See Scenario: Referenceable] for details on
+_jakarta.resource.Referenceable_ . Note that the
+_jakarta.resource.Referenceable_ interface extends the
+_jakarta.naming.Referenceable_ interface. Refer to section
```
[the remark this answers is not in the document]
Comment (bbulk) [reply spec/src/main/asciidoc/Connectors.adoc]: Yep, changed the naming.Referenceable

File: spec/src/main/asciidoc/Connectors.adoc (hunk 1 above)
Comment (smillidge) [reply spec/src/main/asciidoc/Connectors.adoc:4649]: It is SE https://docs.oracle.com/en/java/javase/11/docs/api/java.transaction.xa/javax/transaction/xa/package-summary.html

**Pull request merged**